package.json

```diff
@@ -4,11 +4,15 @@
   "description": "GraphQL Backend for Human Connection",
   "main": "src/index.js",
   "scripts": {
+    "eslint": "eslint src --config .eslintrc.js",
     "test": "echo \"Error: no test specified\" && exit 0",
     "start": "./node_modules/.bin/nodemon --exec babel-node src/index.js",
     "start:debug": "./node_modules/.bin/nodemon --exec babel-node --inspect=0.0.0.0:9229 src/index.js",
-    "db:seed": "concurrently --kill-others --success first 'cross-env GRAPHQL_URI=http://localhost:4001 PERMISSIONS=disabled GRAPHQL_PORT=4001 yarn run start' 'wait-on tcp:4001 && cross-env GRAPHQL_URI=http://localhost:4001 ./node_modules/.bin/babel-node src/seed/seed-db.js'",
-    "db:reset": "concurrently --kill-others --success first 'cross-env GRAPHQL_URI=http://localhost:4001 PERMISSIONS=disabled GRAPHQL_PORT=4001 yarn run start' 'wait-on tcp:4001 && cross-env GRAPHQL_URI=http://localhost:4001 ./node_modules/.bin/babel-node src/seed/unseed-db.js'"
+    "db:no-auth-server": "cross-env GRAPHQL_URI=http://localhost:4001 PERMISSIONS=disabled GRAPHQL_PORT=4001 yarn run start",
```

Comment:
I must admit that this script feels a bit dangerous and should not be run standalone. I think we should put it in the variable section of the package.json

Reply:
@roschaefer could we put the no auth server part into the config part of the package.json?
See: https://frontend.irish/npm-config-variables